SQL/vetclinicData.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN TRANSACTION;
+INSERT INTO "clients" ("client_id","first_name","last_name","phone","email") VALUES (1,'john','doe','555-321-6543','johndoe@mail.com');
+INSERT INTO "pets" ("pet_id","name","species","gender","dob","owner") VALUES (1,'Isis','cat','FS','2006-11-01',1);

```

Comment:
Can we give the cat a different name?
```suggestion
INSERT INTO "pets" ("pet_id","name","species","gender","dob","owner") VALUES (1,'Lily','cat','FS','2006-11-01',1);

```